packages/core/test/lib/envelope.test.ts

```diff
@@ -40,15 +40,20 @@ describe('createEventEnvelope', () => {
           user: { id: 'bob', segment: 'segmentA' },
           transaction: 'TX',
         },
-        'sentry-environment=prod,sentry-release=1.0.0,sentry-transaction=TX,sentry-userid=bob,sentry-usersegment=segmentA',
+        {
+          release: '1.0.0',
+          environment: 'prod',
+          user: { id: 'bob', segment: 'segmentA' },
+          transaction: 'TX',
```

Comment:
Thanks, this schema looks correct now based on the [reference struct](https://github.com/getsentry/relay/blob/b23b8d3b2360a54aaa4d19ecae0231201f31df5e/relay-sampling/src/lib.rs#L685-L707). However, we currently also expect the `trace_id` and the `public_key` of the DSN where the trace originated. Is this just missing in this test?

Reply:
It's missing in the test, can add!